### PR TITLE
Add helper install for installing provider dependencies

### PR DESCRIPTION
**Pull request by petitfou**

### CODE REVIEW

File: launchers/install-full-deps [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+sudo apt install -yy gstreamer1.0-vaapi gstreamer1.0-tools vainfo mpv python3-tinydb python3-jinja2 \
+                     wl-clipboard gnome-screensaver gnome-screenshot ffmpeg
+
+# kivu team PPA
+sudo apt-add-repository -y 'ppa:kivuteam/ppa'
+sudo apt update
+
+# ydotool
+sudo apt install -yy ydotool ydotoold
```
Comment (mckees) [inline launchers/install-full-deps:11]: Is there a reason you use -yy on the installs? The [docs](https://manpages.ubuntu.com/manpages/xenial/man8/apt-get.8.html)  only show -y as a flag, and this installs on my system without the -yy

File: launchers/install-full-deps [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+# gfxi
+sudo apt install -yy gfxi
```
Comment (mckees) [inline launchers/install-full-deps:14]: Same question about -yy here

File: launchers/install-full-deps [hunk 3]
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+sudo apt install -yy gstreamer1.0-vaapi gstreamer1.0-tools vainfo mpv python3-tinydb python3-jinja2 \
```
Comment (mckees) [inline launchers/install-full-deps:3]: Same question about -yy as I asked below

File: launchers/install-full-deps (hunk 1 above)
Comment (petitfou) [reply launchers/install-full-deps]: that is a typo from an old habit, thanks

File: launchers/install-full-deps (hunk 2 above)
Comment (petitfou) [reply launchers/install-full-deps]: that is a typo from an old habit, thanks

File: launchers/install-full-deps (hunk 3 above)
Comment (petitfou) [reply launchers/install-full-deps]: typo, old habit, thanks

**Review by mckees — APPROVED** [review]
LGTM

**Comment by pieqq** [comment]
Is there a reason for putting this setup script into the `launchers` directory?

**Comment by petitfou** [comment]
> Is there a reason for putting this setup script into the `launchers` directory?

Just convenient and also because that in the launchers directory, we are already having scripts like configure or shell-wrapper that are not really launcher

**Pull request merged**